apps/studio/prisma/generated/generatedEnums.ts

```diff
@@ -12,6 +13,23 @@ export const ResourceType = {
   IndexPage: "IndexPage",
 } as const
 export type ResourceType = (typeof ResourceType)[keyof typeof ResourceType]
+||||||| parent of c8c7a8cf (chore: tests)
+    RootPage: "RootPage",
+    Page: "Page",
+    Folder: "Folder",
+    Collection: "Collection",
+    CollectionPage: "CollectionPage"
+} as const;
+export type ResourceType = (typeof ResourceType)[keyof typeof ResourceType];
+=======
+  RootPage: "RootPage",
+  Page: "Page",
+  Folder: "Folder",
+  Collection: "Collection",
+  CollectionPage: "CollectionPage",
+} as const
+export type ResourceType = (typeof ResourceType)[keyof typeof ResourceType]
+>>>>>>> c8c7a8cf (chore: tests)
```

Comment:
issue: I think some rebasing issues here

Reply:
fixed, thanks for pointing it out